src/lib/drivers/accelerometer/PX4Accelerometer.cpp

```diff
@@ -208,6 +257,19 @@ void PX4Accelerometer::update(hrt_abstime timestamp_sample, float x, float y, fl
 
 void PX4Accelerometer::updateFIFO(const FIFOSample &sample)
 {
+	// check for parameter updates
+	if (_parameter_update_sub.updated()) {
+		// clear update
+		parameter_update_s pupdate;
+		_parameter_update_sub.copy(&pupdate);
+
+		// update parameters from storage
+		ModuleParams::updateParams();
+		UpdateCalibration();
+
+		ConfigureFilter(_param_imu_accel_cutoff.get());
+	}
```

Comment:
Since this is duplicated I would extract it in a function and call it `checkForParameterUpdates()`

Reply:
Makes sense. 

By the way, this might be another little reminder that we should hash out a basic style guide. On a number of occasions various people have referenced or fallen back to the google style guide, which is why I've been naming functions and methods mixed case starting with a capital letter. Then getters and setters are named like variables. 

I could probably live with any of the well thought out standards at this point, I mainly want consistency to strive for and perhaps some tooling. https://google.github.io/styleguide/cppguide.html#Naming